src/main/java/seedu/rc4hdb/ui/HelpWindow.java

```diff
@@ -89,6 +92,19 @@ public void focus() {
         getRoot().requestFocus();
     }
 
+    /**
+     * Closes the help window on Escape key press.
+     */
+    private void setAccelerators() {
+        getRoot().addEventHandler(KeyEvent.KEY_PRESSED, (KeyEvent event) -> {
+            if (KeyCode.ESCAPE == event.getCode()) {
+                getRoot().close();
+            }
+            ;
+        }
+        );
+    }
```

Comment:
Looks like it would have some sort of checkstyle error, but somehow it doesn't oO.

I believe the ';' in line 103 is redundant, correct me if I'm wrong.